maro/rl/rollout/env_sampler.py

```diff
@@ -283,8 +283,6 @@ def _get_global_and_agent_state(
             policy = self._policy_dict[policy_name]
             if isinstance(policy, RLPolicy) and not isinstance(state, np.ndarray):
                 raise ValueError(f"Agent {agent_name} uses a RLPolicy but its state is not a np.ndarray.")
-            if not isinstance(policy, RLPolicy) and not isinstance(state, list):
```

Comment:
Why remove this?

Reply:
the action of a specific **agent** can be any object. Here the "state" is not the List of state for a **policy**